docs/acm/component/server-foundation/metrics.md

```diff
@@ -8,10 +8,11 @@ Nomenclature:
+| acm_managed_cluster_status_condition | gauge | Managed cluster status condition | <table> <tr> <th>Number</th> <th>Label</th> <th>Description</th> </tr> <tr> <td>1</td> <td>`managed_cluster_id`</td> <td>The managed cluster ID is the cluster ID if the managed cluster is an {ocp-short} cluster. Otherwise, the managed cluster ID is the cluster name.</td> </tr> <tr> <td>2</td> <td>`managed_cluster_name`</td> <td>The name of the managed cluster.</td> </tr> <tr> <td>3</td> <td>`condition`</td> <td>The name of the condition.</td> </tr> <tr> <td>4</td> <td>`status`</td> <td>The status of the condition. Valid values are `true`, `false`, or `unknown`. For required conditions, such as `ManagedClusterConditionAvailable`, the status is set to unknown if the condition is missing. </td> </tr></table>  | Not exposed |
 | acm_manifestwork_status_condition | gauge | Manifestwork status condition | <table> <tr> <th>Number</th> <th>Label</th> <th>Description</th> </tr> <tr> <td>1</td> <td>`manifestwork`</td> <td>The name of the manifestwork.</td> </tr> <tr> <td>2</td> <td>`managed_cluster_id`</td> <td>The name of the managed cluster to which the manifestwork belongs.</td> </tr> <tr> <td>3</td> <td>`managed_cluster_name`</td> <td>The name of the managed cluster to which the manifestwork belongs.</td> </tr> <tr> <td>4</td> <td>`condition`</td> <td>The name of the condition.</td> </tr> <tr> <td>5</td> <td>`status`</td> <td>The status of the condition. Valid values are `true`, `false`, or `unknown`. For required conditions, such as `Applied` and `Available`, the status is set to unknown if the condition is missing. </td> </tr></table>  | Not exposed |
 | acm_manifestwork_count | - | Manifestwork count | None | Not exposed |
+| acm_managed_cluster_worker_cores | gauge | The core count of worker nodes on the managed cluster. | <table> <tr> <th>Number</th> <th>Label</th> <th>Description</th> </tr> <tr> <td>1</td> <td>`hub_cluster_id`</td> <td>The cluster ID of the hub cluster.</td> </tr> <tr> <td>2</td> <td>`managed_cluster_id`</td> <td>The managed cluster ID is the cluster ID if the managed cluster is an OpenShift Container Platform cluster. Otherwise, the managed cluster ID is a unique ID of the cluster.</td> </tr> </table> | Not exposed, STABLE |
```

Comment:
I'd like to change the ClusterID description of the new metric to
```
The managed cluster ID is the cluster ID if the managed cluster is an OpenShift Container Platform cluster. Otherwise, the managed cluster ID is the cluster name.
```

Reply:
Updated